apollo-router/src/configuration/testdata/migrations/minor/health_check.yaml

```diff
@@ -1,2 +0,0 @@
-health-check:
```

Comment:
I'm a bit confused by this file 🤔  It doesn't look like this change is related to the PR?

Reply:
It's mainly because I changed the test because now I have a minor migration so it changed